assets/scripts/vue/components/signalement-form/components/SignalementFormDisorderOverview.vue

```diff
@@ -104,6 +104,10 @@
       <br>
       <h5 class="fr-h6">Précisions sur la situation</h5>
       <p class="white-space-pre-line">{{ formStore.data[idMessageAdministration] }}</p>
+      <p>
+        <span v-if="formStore.data['message_administration_documents_upload'].length > 0">{{ formStore.data['message_administration_documents_upload'].length }} document(s) complémentaire(s) ajouté(s).<br></span>
```

Comment:
Il me reste un doute.
Quand je vais sur le formulaire avec des fixtures, pour certaines le formulaire ne se charge pas car data.message_administration_documents_upload is undefined.
Donc vérifier ici si data.message_administration_documents_upload est defined. Sinon des utilisateurs pourront avoir un souci en reprenant un vieux brouillon. (et c'est relou pour les fixtures ;) )

![image](https://github.com/user-attachments/assets/625ffbb7-0fe7-4e2c-8338-c77d2d423bc9)